mem-01-b.xml

```diff
@@ -4307,6 +4754,25 @@ The imperative verb prefix when {bIH:n:nolink} is the object is {tI-:v:pref}.</c
 Глагольный префикс повелительной формы, когда {bIH:n:nolink} предмет, в отношении которого производится действие {tI-:v:pref}.</column>
       <column name="notes_zh_HK"></column>
       <column name="notes_pt"></column>
+      <column name="notes_fi">Tämä on monikko {'oH:n:pro}.
+
+Verbien etuliitteet, kun aihe {bIH:n:nolink} on:
+▶ {bIH:n:nolink}: {pagh:n:1h} = {0:v:pref}
```

Comment:
Is there a reason to add a space (after the colon) in Finnish, or is that just an artifact of the machine translation? (Same comment for all pronoun tables.)

Reply:
It's an artifact. I checked all notes_fi fields that had broken links, but others can still have errors.

In general the quality of En->Fi machine translation is very poor for this kind of data, so machine translated notes aren't much of use anyway. I'm looking into translating all notes for the entries in the menu properly in short time.